packages/firebase_messaging/firebase_messaging/android/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
   <uses-permission android:name="android.permission.INTERNET"/>
   <uses-permission android:name="android.permission.WAKE_LOCK"/>
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+  <!-- Permissions options for the `notification` group -->
+  <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
We should probably mention this in the changelog for this PR

Reply:
We can manually edit this when doing the next release I guess 🤔

Reply:
I think you can change the squash message when merging from GitHub directly